### PR TITLE
Send email verification in AWS ses.

### LINKED ISSUE #72: Add email as unverified ses endpoint and send ses confirmation email.
**Todo**

- [x] Connect to ses using BOTO3.
- [x] Add email to ses as an unverified email.
- [x] Send confirmation to email

**Story**
As a developer, letting customers sign up for email verifications will enable greater functionality (email notification), allowing for a better user experience!!

### PR DESCRIPTION
**Pull request by BerkayDur**
Adds the functionality to send ses email verifications programmatically using boto3. This is essential to enabling the functionality for users to receive email notifications on price changes!

Closes #72

### CODE REVIEW

File: Dashboard/utilities/email_verification.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        logging.error('send_verification_email passed `email` argument not of type str.')
+        return {'success': False, 'reason': 'bad email type, email must be of type str.'}
+    if not is_ses(boto_ses_client):
+        logging.error('send_verification_email client is not a boto3 ses client.')
+        return {'success': False, 'reason': 'client is not a boto3 ses client.'}
+    logging.info('Sending email verification...')
+    try:
+        response = boto_ses_client.verify_email_identity(EmailAddress=email)
+    except ClientError as e:
+        if e.response.get('Error').get('Code') == 'InvalidClientTokenId':
```
Comment (A-Faris) [inline Dashboard/utilities/email_verification.py]: Does using 2 .get() methods consecutively work if the first one fails?

File: Dashboard/utilities/email_verification.py
Comment (A-Faris) [inline Dashboard/utilities/email_verification.py:1]: Shows good reasons

File: Dashboard/utilities/test_email_verification.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
+'''contains test files for email_verification.py'''
+from unittest.mock import MagicMock, patch
+
+import pytest
+from botocore.client import BaseClient
+from botocore.exceptions import ClientError
+
+from email_verification import (is_ses, send_verification_email)
```
Comment (A-Faris) [inline Dashboard/utilities/test_email_verification.py:8]:
```suggestion
from email_verification import is_ses, send_verification_email
```

File: Dashboard/utilities/test_email_verification.py
Comment (A-Faris) [inline Dashboard/utilities/test_email_verification.py:1]: Nice tests

**Review by A-Faris — APPROVED** [review]
If the get methods work then great work

File: Dashboard/utilities/email_verification.py (hunk 1 above)
Comment (BerkayDur) [reply Dashboard/utilities/email_verification.py]: It does not. That is a good point, I will look for this explicitly

Comment (BerkayDur) [reply Dashboard/utilities/email_verification.py]: I've just fixed it - it was a really good spot

**Pull request merged**